### PR TITLE
`Conductor.require` and `Conductor.requireCSS` use relative path

### PR DESCRIPTION
**Pull request by cyril-sf**

### CODE REVIEW

**Comment by cyril-sf** [comment]
@hjdivad good for review

File: dist/conductor.js-0.1.0.js [hunk 1]
```diff
@@ -1994,9 +1995,19 @@ define("oasis",
...
           }
         };
-        xhr.open("get", url, true);
+        xhr.open("get", absoluteURL, true);
         xhr.send();
       }
+    },
+
+    absPath: function(url){
+      var Loc = this.sandbox.options.url;
```
Comment (hjdivad) [inline dist/conductor.js-0.1.0.js:2004]: `Loc` should be lowercase, no?

File: dist/conductor.js-0.1.0.js [hunk 2]
```diff
@@ -1994,9 +1995,19 @@ define("oasis",
...
+      Loc = Loc.substring(0, Loc.lastIndexOf('/'));
+      while (/^\.\./.test(url)){
```
Comment (hjdivad) [inline dist/conductor.js-0.1.0.js:2006]: This would probably be clearer with `replace`

File: dist/conductor.js-0.1.0.js [hunk 3]
```diff
@@ -1994,9 +1995,19 @@ define("oasis",
             promise.reject({status: this.status});
           }
         };
-        xhr.open("get", url, true);
+        xhr.open("get", absoluteURL, true);
         xhr.send();
       }
+    },
+
+    absPath: function(url){
```
Comment (hjdivad) [inline dist/conductor.js-0.1.0.js:2003]: This doesn't actually return an absolute path.  It just replaces instances of `..`.  We should rename this function

File: dist/conductor.js-0.1.0.js (hunk 2 above)
Comment (cyril-sf) [inline dist/conductor.js-0.1.0.js:2006]: not sure to see how you'd like to do it with `replace`

File: dist/conductor.js-0.1.0.js (hunk 3 above)
Comment (cyril-sf) [inline dist/conductor.js-0.1.0.js:2003]: `expandPath`?

Comment (hjdivad) [inline dist/conductor.js-0.1.0.js:2003]: seems reasonable.

File: dist/conductor.js-0.1.0.js (hunk 2 above)
Comment (hjdivad) [inline dist/conductor.js-0.1.0.js:2006]: wait what does it even mean to require a resource that contains `..` if the urls are relative to the card location?  why can't we just say that resources that don't begin with `/` are just appended to the sandbox's url?

**Pull request merged**